daemon/service.h

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#ifndef NETDATA_SERVICE_H
+#define NETDATA_SERVICE_H 1
+
+#include "daemon/common.h"
+
+/* Run service jobs every X seconds */
+#define SERVICE_HEARTBEAT 10
```

Comment:
Should this be a user-configurable option?

Reply:
In my opinion no... but it could be variable during runtime....

Reply:
@vlvkobal if you don't have a different opinion on this, you can move the macro definition in `daemon/service.c` as it's not used anywhere else.

Reply:
Same for this :-) We can "pick" a time to run cleanups, maybe when there's not much else going on (cpu side), but in later iterations.

Reply:
I don't think it should be configurable. I moved the macro from the header to the source.